components/speedreader/rust/lib/src/readability/src/extractor.rs

```diff
@@ -1,15 +1,22 @@
-use markup5ever_rcdom::RcDom;
-use markup5ever_rcdom::SerializableHandle;
+use dom;
```

Comment:
I'm actually surprised this works, but I believe it's preferred to be use more explicit imports like `crate::dom`.